.github/workflows/node.js.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Deployment to AWS lambda
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
```

Comment:
This whole `strategy` section can be removed and only keep the `step` below with `node-version: 12.x` if I'm not mistaken..

Reply:
good catch!